packages/visual-editor/package.json

```diff
@@ -84,6 +84,7 @@
       "files": [
         "src/**/*.ts",
         "tests/**/*.ts",
+        ".env*",
```

Comment:
I think we can move this to the `files` of `build:vite`, since it's only needed there. No need to invalidate cached typescript builds with this file, since `tsc` isn't reading it.

Reply:
done